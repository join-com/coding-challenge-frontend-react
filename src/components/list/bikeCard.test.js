import React from 'react';
import { shallow , mount} from 'enzyme';
import BikeCard from './bikeCard';

describe('BikeCard', () => {
    it('it should render correctly', () => {
        const component = shallow(<BikeCard />);
        expect(component).toMatchSnapshot();
    });
})