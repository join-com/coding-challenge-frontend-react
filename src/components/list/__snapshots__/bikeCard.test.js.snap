// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BikeCard it should render correctly 1`] = `
<BikeCard
  classes={
    Object {
      "cardFooterContent": "BikeCard-cardFooterContent-6",
      "dateRangeIcon": "BikeCard-dateRangeIcon-5",
      "gridPad": "BikeCard-gridPad-3",
      "gridPadInner": "BikeCard-gridPadInner-4",
      "imgHover": "BikeCard-imgHover-7",
      "main": "BikeCard-main-1",
      "searchBtn": "BikeCard-searchBtn-2",
      "searchIcon": "BikeCard-searchIcon-8",
      "title": "BikeCard-title-9",
    }
  }
/>
`;
